tex/chProofs.tex

```diff
@@ -499,9 +527,9 @@ \subsection{Proofs by computation}\label{ssec:proofcomp}
 \end{coq}
 \coqrun{name=secondp}{ssr,secondp,verbose,qed}
 
-Indeed, this statement holds because the two hand sides of the equality
+Indeed, this statement holds because the two sides of the equality
 are the same, once the definition of the \C{addn} function, hidden
-behind the infix \C{+} notation, is unfolded, and that the calculation
+behind the infix \C{+} notation, is unfolded, and once the calculation
```

Comment:
The problem is that "the calculation is performed" would look like a main clause if I remove the "once".